test/e2e/suite/issuers/ca/certificaterequest.go

```diff
@@ -120,6 +120,23 @@ var _ = framework.CertManagerDescribe("CA CertificateRequest", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 
+		It("should be able to obtain an Ed25519 key from a RSA backed issuer", func() {
```

Comment:
Can we add a test case to the e2e conformance suite? This will ensure it is run against all issuer types without needing to implement a test in all these different places (and allows any external issuers to also validate their behaviour with this keyAlgorithm)

Reply:
I took a quick look before my computer ran out of battery...

I couldn't figure it out as easily, you have any commits that have implemented it in the past so I can just look at the different portions of the commit and learn from that?

Reply:
I'm not super familiar with this part of the code base and I'm about to leave to start my weekend, but I think you could start by pretty much copy-pasting this chunk [here](https://github.com/jetstack/cert-manager/blob/5e2a6883c1202739902ac94b5f4884152b810925/test/e2e/suite/conformance/certificates/tests.go#L82-L108) and replacing ECDSA with Ed25519

That would be a great start.

Reply:
I think it worked. Going to wait for @munnerz  to confirm.